app/main.py

```diff
@@ -1,6 +1,16 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.environ["WEATHER_API_KEY"]
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    url = f"https://api.weatherapi.com/v1/current.json?key={API_KEY}=Paris"
+    response = requests.get(url)
```

Comment:
Use `params` instead of `?key={API_KEY}=Paris`
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls